message/utils/checkIfEmpty.js

```diff
@@ -0,0 +1,15 @@
+const checkIfEmpty = (str) => {
+
+    if (str === '') return true
+
+    const strArr = str.split('')
+
+    for (let i = 0; i < strArr.length; i++) {
```

Comment:
@wablopilson  Can you please expalin the usecase that you are trying solve here?

Reply:
I created that util to check if any string you send to that function coming from a request (in our case, message or destination) is empty or is just filled with spaces.